Basic-Car-Maintenance.xcodeproj/project.pbxproj

```diff
@@ -100,7 +106,6 @@
 		E58499692ACDDAFF00634660 /* Contributor.swift */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.swift; path = Contributor.swift; sourceTree = "<group>"; };
 		FF0813562AD0A83000910EFA /* UITests.xcconfig */ = {isa = PBXFileReference; lastKnownFileType = text.xcconfig; path = UITests.xcconfig; sourceTree = "<group>"; };
 		FF0813572AD0A92700910EFA /* Widget.xcconfig */ = {isa = PBXFileReference; lastKnownFileType = text.xcconfig; path = Widget.xcconfig; sourceTree = "<group>"; };
-		FF098EFA2AB3424E003EC0FE /* Basic-Car-Maintenance.xcconfig */ = {isa = PBXFileReference; lastKnownFileType = text.xcconfig; path = "Basic-Car-Maintenance.xcconfig"; sourceTree = SOURCE_ROOT; };
```

Comment:
@DandyLyons Can you go through and make sure this is added here?
Sorry I misspoke earlier the `Basic-Car-Maintenance.xcconfig` is ignored in the `.gitignore`, but it's still used in Xcode, because that's how it's used in the project

I think you should be able to use any text editor, and add these lines back, just make sure they're in the right place

Reply:
Done.